docs/reference/search/profile.asciidoc

```diff
@@ -163,7 +163,38 @@ The API returns the following result:
             ]
           }
         ],
-        "aggregations": []
+        "aggregations": [],
+        "fetch": {
+          "type": "fetch",
+          "description": "fetch",
```

Comment:
This `type` and `description` come from my using a raw `ProfileResult` for the entire fetch phase. I'm not happy about them, but I wasn't sure if it was worth writing a whole top level class for the result or working around them some other way.